### PR TITLE
Vne/feature add dockerfile

### PR DESCRIPTION
**Pull request by vladimir-nechai**
- add nginx conf
- add Dockerfile
- refactor name `banking-gateway-ui` to `fintech-ui`

### CODE REVIEW

**Comment by DG0lden** [comment]
pls think about adding build and tests run into pipeline

**Comment by DG0lden** [comment]
Could you please move `fintech-ui` under `fintech-examples` ?

**Comment by saschanuissl** [comment]
@DG0lden Sure, that's done. If @collins-k needs help for his open PR after moving fintech-ui, reach out.

File: fintech-examples/fintech-ui/entry.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+export DOLLAR='$'
```
Comment (DG0lden) [inline fintech-examples/fintech-ui/entry.sh]: You don't need this line if you use erb, don't you?

Comment (saschanuissl) [reply fintech-examples/fintech-ui/entry.sh]: You're right. Done.

**Pull request merged**